README.md

```diff
@@ -135,22 +137,14 @@ echo 8 > /sys/class/net/enp2s0f0/device/sriov_numvfs
 ### Main parameters
 * `name` (string, required): the name of the network
 * `type` (string, required): "sriov"
-* `master` (string, required): name of the PF
-* `l2enable` (boolean, optional): if `true` then add VF as L2 mode only, IPAM will not be executed
+* `deviceID` (string, required): A valid pci address of an SRIOV NIC's VF. e.g. "0000:03:02.3"
 * `vlan` (int, optional): VLAN ID to assign for the VF
 * `ipam` (dictionary, optional): IPAM configuration to be used for this network.
-* `dpdk` (dictionary, optional): DPDK configuration
-
-### Using DPDK drivers:
-If this plugin is used to bind a VF to dpdk driver inside the container, then the IPAM configuration will be ignored.
+* `dpdkMode` (bool, optional): `true` Indicates whether VF is attached with DPDK enabled driver. Default is `false`
```

Comment:
So, this should be auto detected and user will not need to specify ?

Reply:
For Mellanox cards, it seems not detectable whether the interface is in SR-IOV or DPDK mode, in which case I think it shall be fine not using dpdk mode option. @moshe010 can you help to confirm? for example, there shall be no additional dpdk-specific configuration needed in CNI in order to use dpdk mode.

Reply:
I confirm :)